src/stdatamodels/jwst/datamodels/schemas/combinedspec.schema.yaml

```diff
@@ -15,6 +15,12 @@ allOf:
       default: 1
       fits_keyword: SPORDER
       fits_hdu: COMBINE1D
+    dispersion_direction:
+      title: Dispersion direction
+      type: integer
+      fits_keyword: DISPAXIS
+      fits_hdu: COMBINE1D
+      default: -999
```

Comment:
What does "-999" do here and why not leave off the default (if it's a nonsense value)? Also, is the reasoning for this "-999" the same as for the other "-999" in the other schemas for this PR?

Reply:
I think you are right that these are never used and should be removed where they have been added for metadata.  In fact, it may be worth considering whether to remove the default for exposure times and things too.

The defaults are used nontrivially when they exist in the spec_table attributes, because they populate the table where there is no data.  But here, there's either a global value for the whole dataset, or there isn't.

Reply:
not special I just missed it somehow, glad you checked again